prow/monitoring/BUILD.bazel

```diff
@@ -56,3 +62,30 @@ k8s_objects(
       ":servicemonitor_crd",
    ]
 )
+
+load("//def:configmap.bzl", "k8s_configmap")
+
+k8s_configmap(
+    name = "grafana-datasources",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    namespace = "prow-monitoring",
+    data = {
+        "datasources.yaml": ":datasources.yaml",
+    },
+)
+
+k8s_objects(
+   name = "monitoring-config",
+   objects = [
+      ":grafana-datasources",
+   ]
+)
+
+### workaround before we have k8s_secret at hand
```

Comment:
@fejta is there a tracking issue for this one? I didn't keep track of that conversation

Reply:
https://github.com/kubernetes/test-infra/issues/13000